src/SIL.LCModel.Core/SIL.LCModel.Core.csproj

```diff
@@ -226,43 +226,43 @@
     <None Include="Phonology\PhonEnvError.dtd" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="..\..\lib\downloads\x86\icudt54.dll" Condition="'$(Platform)'=='x86' and '$(OS)'=='Windows_NT'">
+    <Content Include="..\..\lib\downloads\x86\icudt54.dll" Condition="'$(Platform)'=='x86'">
       <Link>icudt54.dll</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="..\..\lib\downloads\x86\icuin54.dll" Condition="'$(Platform)'=='x86' and '$(OS)'=='Windows_NT'">
+    <Content Include="..\..\lib\downloads\x86\icuin54.dll" Condition="'$(Platform)'=='x86'">
       <Link>icuin54.dll</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="..\..\lib\downloads\x86\icutu54.dll" Condition="'$(Platform)'=='x86' and '$(OS)'=='Windows_NT'">
+    <Content Include="..\..\lib\downloads\x86\icutu54.dll" Condition="'$(Platform)'=='x86'">
       <Link>icutu54.dll</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="..\..\lib\downloads\x86\icuuc54.dll" Condition="'$(Platform)'=='x86' and '$(OS)'=='Windows_NT'">
+    <Content Include="..\..\lib\downloads\x86\icuuc54.dll" Condition="'$(Platform)'=='x86'">
       <Link>icuuc54.dll</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="..\..\lib\downloads\x64\icudt54.dll" Condition="'$(Platform)'!='x86' and '$(OS)'=='Windows_NT'">
+    <Content Include="..\..\lib\downloads\x64\icudt54.dll" Condition="'$(Platform)'!='x86'">
       <Link>icudt54.dll</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="..\..\lib\downloads\x64\icuin54.dll" Condition="'$(Platform)'!='x86' and '$(OS)'=='Windows_NT'">
+    <Content Include="..\..\lib\downloads\x64\icuin54.dll" Condition="'$(Platform)'!='x86'">
       <Link>icuin54.dll</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="..\..\lib\downloads\x64\icutu54.dll" Condition="'$(Platform)'!='x86' and '$(OS)'=='Windows_NT'">
+    <Content Include="..\..\lib\downloads\x64\icutu54.dll" Condition="'$(Platform)'!='x86'">
       <Link>icutu54.dll</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="..\..\lib\downloads\x64\icuuc54.dll" Condition="'$(Platform)'!='x86' and '$(OS)'=='Windows_NT'">
+    <Content Include="..\..\lib\downloads\x64\icuuc54.dll" Condition="'$(Platform)'!='x86'">
```

Comment:
On Linux the icu libraries are actually used from a system path. So please add back the OS condition for the icu.dlls.
After that this will be ready to merge! Good work!